### ISSUE TITLE
dev libraries in pipfile

### ISSUE THREAD

**Issue by ezzra** [shacker/django-todo#49]
pipfile is currently:

```
[packages]
django = "*"
django-extensions = "*"
"psycopg2-binary" = "*"
pytest = "*"
pytest-django = "*"
"flake8" = "*"
factory-boy = "*"
titlecase = "*"
bleach = "*"

[dev-packages]
pylint = "*"
mypy = "*"
```

shouldn't `pytest`, `pytest-django`, `flake8`, `factory-boy` be in `[dev]` ?

**Comment by shacker** [comment]
Oops yes - will fix, thanks.

**Comment by shacker** [comment]
Fixed in d0212b8a55a6dd87c80fa5d4627879a75875b04a
